xml_converter/src/xml_converter.cpp

```diff
@@ -158,19 +172,22 @@ void process_data(
     // All of the loaded pois and categories
     vector<Parseable*> parsed_pois;
     map<string, Category> marker_categories;
-    set<string> top_level_categories;
-    vector<string> duplicate_categories;
+    map<string, vector<vector<string>>> category_to_marker_pack_names;
+    map<string, vector<string>> duplicate_categories;
 
     // Read in all the xml taco markerpacks
     auto begin = chrono::high_resolution_clock::now();
     for (size_t i = 0; i < input_taco_paths.size(); i++) {
         cout << "Loading taco pack " << input_taco_paths[i] << endl;
 
-        set<string> category_names = read_taco_directory(
+        // swap this to map <category, set<filenames>>)
```

Comment:
Is this something you wanted to do in this pr? If not why not?

Reply:
This was a working comment. It is now the way i wanted but missed removing the comment